packages/fhir-client/src/components/PatientListView/tests/index.test.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { MemoryRouter } from 'react-router-dom';
+import { PatientListView } from '..';
+
+describe('PatientListView', () => {
+  let queryClient;
+
+  beforeEach(() => {
+    queryClient = new QueryClient();
+  });
+
+  afterEach(() => {
+    queryClient.clear();
+  });
+
```

Comment:
Here is an [example](https://github.com/onaio/fhir-web/blob/main/packages/fhir-care-team/src/components/ListView/tests/index-rtl.test.tsx) of how to setup the test suites. You'll want to setup :
1. nock for api call mocking.
2. One can create a wrapper HOC that dry's out the providers out.
3. How to initialize the query client for tests so there are minimal suprises.

It takes a bit of a learning curve to especially debug test issues, so try it out and let me know if you would like to pair on this.